open-bot.yaml

```diff
@@ -85,10 +85,11 @@ rules:
       identifier: "ci-result"
       message: |-
         @{{commit.author.login}} Please review the following output log for errors:
```

Comment:
You could try only showing the first error and remaining in details with this `filter`:

```
string_cleanup:
  id: firstError
  value: "{{{logResult}}}"
  remove:
    - "\\s2\\)\\s[\\s\\S]*"
string_cleanup:
  id: remainingErrors
  value: "{{{logResult}}}"
  remove:
    - "^[\\s\\S]+?(?=\\s2\\)\\s)"
```

Try the regexps, I haven't...